### ISSUE TITLE
Do you want an update of JupyterLab 2 to JupyterLab 3, and misc packages?

### ISSUE THREAD

**Issue by consideRatio** [learning-2-learn/jhub-deployment#88]
We are using a base image that has been pinned to a version of JupyterLab 2 for quite a while, this can be updated to one with JupyterLab 3 - which was problematic mainly a while ago when a lot of packages hadn't added support for JupyterLab 3. Now though, these packages typically work well again.

@arokem @DetectiveKiwi, do you want me to bump to the latest base image and get misc updates, for example JupyterLab 3?

**Comment by DetectiveKiwi** [comment]
Sounds good to me!

**Comment by arokem** [comment]
Yeah - that would be great! Thanks!

**Comment by consideRatio** [comment]
I've made the update, and after pressing merge I've also realized I bumped Python 3.8 to 3.9 without asking if that was okay first. Let me know if you run into issues and if that was an acceptable upgrade or not.

It is upgraded in the base image we use, and I think if we must not upgrade to Python 3.9 we should use an older version of the base image - preferably with JupyterLab 3 still.

See #89 for the concrete changes.
